tests/lib/assert_helper.js

```diff
@@ -363,6 +363,8 @@ PresenceMessage.prototype.assert_ack_for = function(type, message) {
   var expected = { to: this.scope };
   var ackNumber = message.ack;
   delete message.ack;
+  var options = message.options;
+  delete message.options;
```

Comment:
Don't know what this change is, but best to avoid it.. I am expecting the changes to pass with no changes to tests in radar ideally.